apps/idos-enclave/src/lib/enclave.js

```diff
@@ -48,10 +48,10 @@ export class Enclave {
     });
   }
 
-  storage(humanId, signerAddress, signerPublicKey, expectedUserEncryptionPublicKey) {
+  storage(humanId, signerAddress, signerEncryptionPublicKey, expectedUserEncryptionPublicKey) {
```

Comment:
`signerEncryptionPublicKey` doesn't make sense. A signer is a crypto device used for auth.

What am I missing?

Reply:
``userEncryptionPublicKey`` ?